Web/.env.production

```
@@ -0,0 +1,19 @@
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_cHJlY2lvdXMtaGFyZS00NC5jbGVyay5hY2NvdW50cy5kZXYk
+CLERK_FRONTEND_API_URL=https://precious-hare-44.clerk.accounts.dev
+
+# Convex Authentication Configuration
+# These variables are required for Convex to authenticate with Clerk in production
+CLERK_JWT_ISSUER_DOMAIN=https://precious-hare-44.clerk.accounts.dev
+CONVEX_APPLICATION_ID=convex
+
+# Server-side environment variables
+# These variables are NOT accessible to client-side code
+CLERK_SECRET_KEY=sk_test_13Inigx4tdzibOzMjTPQ91i1KUaGm7eSevPvsA0xZo
+CLERK_WEBHOOK_SECRET=whsec_xIoRACj4xOKG+B349MCfNHCSxToWSIGg
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)